nebula-logger/core/main/log-management/lwc/logViewer/__tests__/logViewer.test.js

```diff
@@ -6,6 +6,26 @@ const MOCK_GET_LOG = require('./data/LogViewerController.getLog.json');
 
 document.execCommand = jest.fn();
 
+jest.mock(
+    'lightning/platformResourceLoader',
+    () => {
+        return {
+            loadScript() {
+                return new Promise((resolve, _) => {
+                    global.Prism = require('../../../staticresources/LoggerResources/prism.js');
+                    resolve();
+                });
+            },
+            loadStyle() {
+                return new Promise((resolve, _) => {
```

Comment:
seems like this one could just be `Promise.resolve()`, no?

Reply:
...that's way better! I'll update this.